testrunner.Makefile

```diff
@@ -0,0 +1,20 @@
+ifndef __TESTRUNNER_MAKEFILE__
+
+__TESTRUNNER_MAKEFILE__ := included
+
+
+makefile_dir := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+include $(makefile_dir)/common.Makefile
+
+
+.build/testrunner: | .build/tmp
+	$(call print_target)
+	cd $(MARKETPLACE_TOOLS_PATH)/testrunner \
+	  && bazel build //runner:main \
+	  && rm -rf ../.build/tmp/testrunner \
+	  && mkdir -p ../.build/tmp/testrunner \
+	  && cp bazel-bin/runner/main ../.build/tmp/testrunner/testrunner \
```

Comment:
this relies on the fact that `.build` is one directory above `$MARKETPLACE_TOOLS_PATH`, which isn't always the case. Split the command at this line:
- `cd` and `bazel build`
- `cp $(MARKETPLACE_TOOLS_PATH)/testrunner/bazel-bin/runner/main .build/tmp/...` and `docker build`

Reply:
I've done this changes. Could you please verify that this is what you expected?

I'm sending in current, but if something is missing, please ping me by email, I will change this.

Thanks!